src/python/pants/init/util.py

```diff
@@ -27,12 +30,30 @@ def init_workdir(global_options: OptionValueContainer) -> str:
     workdir_base = global_options.pants_physical_workdir_base
     workdir_dst = os.path.join(workdir_base, safe_filename_from_path(workdir_src))
 
+    def maybe_create_source_control_symlink():
```

Comment:
I think that this behavior should be behind an option, and disabled by default. In v2, or with --chroot enabled in v1, accessing git would not work anyway. And it's definitely a hermiticity leak to do so.

Reply:
Yeah, long term pants needs a way to provide some version information to tasks / rules, but this is a stop gap for node targets which are allowed to do whatever they want when run, e.g run script with a name.

> I think that this behavior should be behind an option, and disabled by default. In v2, or with --chroot enabled in v1, accessing git would not work anyway

That is reasonable.